src/EditorFeatures/Core.Cocoa/NavigationCommandHandlers/FindDerivedSymbolsCommandHandler.cs

```diff
@@ -91,37 +91,38 @@ private static async Task<IEnumerable<ISymbol>> GatherSymbolsAsync(ISymbol symbo
             {
                 using var token = _asyncListener.BeginAsyncOperation(nameof(FindDerivedSymbolsAsync));
 
-                var context = presenter.StartSearch(EditorFeaturesResources.Navigating, supportsReferences: true, cancellationToken);
+                var (context, combinedCancellationToken) = presenter.StartSearch(EditorFeaturesResources.Navigating, supportsReferences: true, cancellationToken);
+                cancellationToken = combinedCancellationToken;
```

Comment:
💭 I believe this might now be supported?

```suggestion
                (var context, cancellationToken) = presenter.StartSearch(EditorFeaturesResources.Navigating, supportsReferences: true, cancellationToken);
```

Reply:
ah, i'll try that.  i hate it.  but i hate it less than the current code.

Reply:
much nicer.  thanks!